### PR TITLE
[MIG][12.0] website_odoo_debranding

### PR DESCRIPTION
**Pull request by karan-dreambits**

### CODE REVIEW

File: website_odoo_debranding/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Tecnativa, S.L. - Vicent Cubells
+# Copyright 2018 Dreambits Technologies Pvt. Ltd., - Karan Shah
```
Comment (simahawk) [inline website_odoo_debranding/__manifest__.py]: usually we don't change copyright w/ significant contrib to the code. 
You could add yourself to the list of contributors.

File: website_odoo_debranding/README.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Usage
+=====
+
+#. Go to your home page.
+#. Enable or disable *Customize > Footer Copyright > Remove Odoo Promotional
+   Link* at your will. It should be enabled by default.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/186/11.0
```
Comment (simahawk) [inline website_odoo_debranding/README.rst]: this is wrong, but see my comment about readme refactoring

**Review by simahawk — CHANGES_REQUESTED** [review]
thanks for your contrib!
Please, respect (C) and move to new README structure https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

**Comment by simahawk** [comment]
build is red, likely due a change in original view, see https://runbot3.odoo-community.org/runbot/static/build/3340146-516-e89d74/logs/job_20_test_all.txt

File: website_odoo_debranding/__manifest__.py (hunk 1 above)
Comment (karan-dreambits) [reply website_odoo_debranding/__manifest__.py]: WIll fix it :)

File: website_odoo_debranding/README.rst (hunk 2 above)
Comment (karan-dreambits) [reply website_odoo_debranding/README.rst]: Sure, Updating it

**Comment by karan-dreambits** [comment]
Okay, I realized I had checkout saas-12 which still has the old view.
Switching to saas-11.5 now, making changes and checking again

**Comment by karan-dreambits** [comment]
Travis failing due to unavailability of branch 12.0 of odoo

Downloading Odoo
Cloning into '/home/travis/odoo-12.0'...
warning: Could not find remote branch 12.0 to clone.
fatal: Remote branch 12.0 not found in upstream origin
The command "travis_install_nightly" failed and exited with 128 during .
Your build has been stopped.

**Comment by pedrobaeza** [comment]
Yeah, that's expected. Up to the final release of v12, we can only rely on OCA/OCB build stage.

**Review by simahawk — CHANGES_REQUESTED** [review]
@karan-dreambits thanks! LG. Please squash migration commits.

**Comment by karan-dreambits** [comment]
@simahawk Rebased finally

**Review by ThomasBinsfeld — APPROVED** [review]

File: website_odoo_debranding/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+=================================
```
Comment (SimoRubi) [inline website_odoo_debranding/readme/DESCRIPTION.rst]: The title is not needed here, it is generated when you execute https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py

**Review by SimoRubi — CHANGES_REQUESTED** [review]
Thanks for your contribution!
Code review and tested on runbot, just  note about the README

File: website_odoo_debranding/readme/DESCRIPTION.rst (hunk 3 above)
Comment (karan-dreambits) [reply website_odoo_debranding/readme/DESCRIPTION.rst]: Okay @SimoRubi  but there is difference in this Description and manifest one so I will update the manifest one to match this

**Review by simahawk — APPROVED** [review]
LG

**Comment by simahawk** [comment]
@SimoRubi ping :)

**Review by SimoRubi — APPROVED** [review]
Looks good

**Pull request merged**